k8s/upgrade/src/helm/values.rs

```diff
@@ -731,28 +729,11 @@ async fn safe_crd_install(upgrade_values_filepath: &Path, yq: &YqV4) -> Result<(
         YamlKey::try_from(".crds.jaeger.enabled")?,
     );
 
-    let crds_api = KubeClient::crds_api().await?;
-    let mut all_crd_names: Vec<String> = Vec::with_capacity(KUBE_API_PAGE_SIZE as usize);
-    let mut list_params = ListParams::default().limit(KUBE_API_PAGE_SIZE);
-
-    loop {
-        let crd_list = crds_api
-            .list_metadata(&list_params)
-            .await
-            .context(ListCrds)?;
-
-        all_crd_names = all_crd_names
-            .into_iter()
-            .chain(crd_list.iter().map(|metadata| metadata.name_unchecked()))
-            .collect();
-
-        match crd_list.metadata.continue_ {
-            Some(token) => {
-                list_params = list_params.continue_token(token.as_str());
-            }
-            None => break,
-        }
-    }
+    let all_crd_names: Vec<String> = KubeClient::list_crds_metadata()
```

Comment:
Does this paginate?

Reply:
This seems to list all crds, is that the intent?
Guess we don't expect there to be too many crds ?

Reply:
>Does this paginate
@abhilashshetty04 Yes, it does, it uses `paginated_list_metadata`.